projects/core/components/hosted-dropdown/hosted-dropdown.component.ts

```diff
@@ -105,6 +129,13 @@ export class TuiHostedDropdownComponent implements TuiFocusableElementAccessor {
     @Input()
     canOpen = true;
 
+    readonly focused$ = this.focus$.pipe(
+        startWith(false),
+        map(() => this.focused),
+        distinctUntilChanged(),
+        takeUntil(this.destroy$),
```

Comment:
This is unnecessary since you do not subscribe.

Reply:
fixed